homeassistant/components/template/strings.json

```
@@ -397,6 +398,44 @@
         },
         "title": "Template switch"
       },
+      "update": {
+        "data": {
+          "device_id": "[%key:common::config_flow::data::device%]",
+          "device_class": "[%key:component::template::common::device_class%]",
+          "name": "[%key:common::config_flow::data::name%]",
+          "installed_version": "[%key:component::update::entity_component::_::state_attributes::installed_version::name%]",
+          "latest_version": "[%key:component::update::entity_component::_::state_attributes::latest_version::name%]",
+          "install": "Actions on install",
+          "in_progress": "[%key:component::update::entity_component::_::state_attributes::in_progress::name%]",
+          "release_summary": "[%key:component::update::entity_component::_::state_attributes::release_summary::name%]",
+          "release_url": "[%key:component::update::entity_component::_::state_attributes::release_url::name%]",
+          "title": "[%key:component::update::entity_component::_::state_attributes::title::name%]",
+          "update_percent": "Update percent"
+        },
+        "data_description": {
+          "device_id": "[%key:component::template::common::device_id_description%]",
+          "installed_version": "Defines a template to get the installed version.",
+          "latest_version": "Defines a template to get the latest version.",
+          "install": "Defines actions to run when the update is installed.  Receives varables `version` and `backup`",
+          "in_progress": "Defines a template to get the in progress status.",
```

Needs a hyphen. Perhaps also "state" instead of "status"?

```suggestion
          "in_progress": "Defines a template to get the in-progress status.",
```